addons/hr_expense/tests/test_expenses.py

```diff
@@ -148,23 +148,39 @@ def test_expense_sheet_paid_company(self):
             'employee_id': self.expense_employee.id,
             'accounting_date': '2021-01-01',
             'payment_method_line_id': self.outbound_payment_method_line.id,
-            'expense_line_ids': [(0, 0, {
-                'name': 'Car Travel Expenses',
-                'employee_id': self.expense_employee.id,
-                'product_id': self.product_c.id,
-                'total_amount': 350.00,
-                'payment_mode': 'company_account',
-            })]
+            'expense_line_ids': [
+                Command.create({
+                    'name': 'Car Travel Expenses',
+                    'employee_id': self.expense_employee.id,
+                    'product_id': self.product_c.id,
+                    'total_amount': 350.00,
+                    'payment_mode': 'company_account',
+                    'date': '2024-01-01',
+                }),
+                Command.create({
+                    'name': 'Lunch expense',
+                    'employee_id': self.expense_employee.id,
+                    'product_id': self.product_c.id,
+                    'total_amount': 90.00,
+                    'payment_mode': 'company_account',
+                    'date': '2024-01-12',
+                }),
+            ]
         })
 
         expense_sheet.action_submit_sheet()
         expense_sheet.action_approve_expense_sheets()
         expense_sheet.action_sheet_move_create()
 
-        move = expense_sheet.account_move_ids
+        move_twelve_january, move_first_january = expense_sheet.account_move_ids
+
+        self.assertEqual(move_twelve_january.date, fields.Date().to_date('2024-01-12'), 'move date should be the same as the expense date')
+        self.assertEqual(move_first_january.date, fields.Date().to_date('2024-01-01'), 'move date should be the same as the expense date')
```

Comment:
```suggestion
        self.assertEqual(move_twelve_january.date, fields.Date.to_date('2024-01-12'), 'move date should be the same as the expense date')
        self.assertEqual(move_first_january.date, fields.Date.to_date('2024-01-01'), 'move date should be the same as the expense date')
```
to_date is static, so no need to initialise anything :)